src/sentry/runner/commands/run.py

```diff
@@ -777,17 +777,35 @@ def handler(signum, frame):
 @run.command("ingest-replay-recordings")
 @log_options()
 @configuration
-@kafka_options("ingest-replay-recordings")
+@kafka_options(
+    consumer_group="ingest-replay-recordings",
+    include_batching_options=True,
+    default_max_batch_size=20,
+)
 @click.option(
     "--topic", default="ingest-replay-recordings", help="Topic to get replay recording data from"
 )
+@click.option("--processes", default=1, type=int)
+@click.option("--input-block-size", type=int, default=DEFAULT_BLOCK_SIZE)
+@click.option("--output-block-size", type=int, default=DEFAULT_BLOCK_SIZE)
 def replays_recordings_consumer(**options):
     from sentry.consumers import print_deprecation_warning
 
     print_deprecation_warning("ingest-replay-recordings", options["group_id"])
     from sentry.replays.consumers import get_replays_recordings_consumer
 
-    consumer = get_replays_recordings_consumer(**options)
+    consumer = get_replays_recordings_consumer(
```

Comment:
is this backwards compatible with the current way our consumer gets ran in production? (i.e. all of the new args have defaults i assume)?

Reply:
Yes, it should be but I will re-review to be sure.

Reply:
This is backwards compatible.  We should be able to deploy without any ops changes.